src/xrCore/LocatorAPI.cpp

```diff
@@ -416,7 +417,7 @@ void CLocatorAPI::LoadArchive(archive& A, pcstr entrypoint)
             int count = sscanf(read_path.c_str(), "%[^\\]s", alias_name);
             R_ASSERT2(count == 1, read_path.c_str());
 
-            auto P = m_paths.find(alias_name);
+            // auto P = m_paths.find(alias_name);
 
```

Comment:
Looked at the vanilla code, this variable was used in the past. Look like, I forgot to remove it after refactoring. Let's just remove it :)
```suggestion
```